### PR TITLE
Feature/gha docker build

### PR DESCRIPTION
**Pull request by korbelm**

### CODE REVIEW

File: .github/workflows/publish-docker-image.yml [hunk 1]
```diff
@@ -21,132 +30,64 @@ jobs:
...
-#        if: github.ref == 'refs/heads/develop'
-#        run: echo "TIMESTAMP=-$(date +%Y.%m.%d)" >> $GITHUB_ENV
-      - name: Libs
+      - name: Prepare Libs
         run: |
-          cd  ${GITHUB_WORKSPACE}/deploy/lib
-          wget https://download.oracle.com/otn-pub/otn_software/jdbc/233/ojdbc11.jar
-          wget https://jdbc.postgresql.org/download/postgresql-42.7.1.jar
-      - name: Get Powerauth war files
+          wget --no-verbose https://download.oracle.com/otn-pub/otn_software/jdbc/233/ojdbc11.jar -O ${GITHUB_WORKSPACE}/deploy/lib/ojdbc11.jar
```
Comment (zcgandcomp) [inline .github/workflows/publish-docker-image.yml:31]: dáváme tam as of now ojdbc?

File: .github/workflows/publish-docker-image.yml [hunk 2]
```diff
@@ -21,132 +30,64 @@ jobs:
...
+          wget --no-verbose https://jdbc.postgresql.org/download/postgresql-42.7.1.jar -O ${GITHUB_WORKSPACE}/deploy/lib/postgresql.jar
```
Comment (zcgandcomp) [inline .github/workflows/publish-docker-image.yml]: postgresql.jar už je přímo ve warkách v této verzi.

File: .github/workflows/publish-docker-image.yml [hunk 3]
```diff
@@ -21,132 +30,64 @@ jobs:
...
-#      - name: Log in to Azure registry
-#        if: ${{ github.event_name == 'workflow_dispatch' }}
-#        uses: docker/login-action@v2
-#        with:
-#          registry: https://powerauth.azurecr.io/
-#          username: ${{ secrets.ACR_USERNAME }}
-#          password: ${{ secrets.ACR_PASSWORD }}
-      - name: "powerauth-server: Build and push container image to Azure registry"
-        uses: docker/build-push-action@v3
+          wget --no-verbose https://${{ env.INTERNAL_USERNAME }}:${{ env.INTERNAL_PASSWORD }}@wultra.jfrog.io/artifactory/internal-maven-repository/io/getlime/security/powerauth-admin/${{ inputs.version }}/powerauth-admin-${{ inputs.version }}.war -O ${GITHUB_WORKSPACE}/deploy/images/powerauth-admin.war
```
Comment (zcgandcomp) [inline .github/workflows/publish-docker-image.yml]: wget from github itself? It is publicly available.

File: .github/workflows/publish-docker-image.yml [hunk 4]
```diff
@@ -21,132 +30,64 @@ jobs:
...
+        uses: docker/login-action@v3
+        with:
+          registry: https://powerauth.azurecr.io/
+          username: ${{ secrets.ACR_USERNAME }}
+          password: ${{ secrets.ACR_PASSWORD }}
+      - name: "powerauth-${{ matrix.app }}-postgresql: Build and push docker image to Azure registry (if enabled)"
+        uses: docker/build-push-action@v5
         with:
-          push: false
+          push: ${{ inputs.push_to_acr }}
```
Comment (zcgandcomp) [inline .github/workflows/publish-docker-image.yml]: sometimes this is evaluated as string - if this work ok, but `inputs. push_to_acr == true` might be safer..

**Review by zcgandcomp — CHANGES_REQUESTED** [review]

File: .github/workflows/publish-docker-image.yml (hunk 1 above)
Comment (korbelm) [reply .github/workflows/publish-docker-image.yml:31]: ano

File: .github/workflows/publish-docker-image.yml (hunk 2 above)
Comment (korbelm) [reply .github/workflows/publish-docker-image.yml]: odebrano

File: .github/workflows/publish-docker-image.yml (hunk 3 above)
Comment (korbelm) [reply .github/workflows/publish-docker-image.yml]: switched

**Review by zcgandcomp — APPROVED** [review]
Looks Ok.

**Pull request merged**